CMakeLists.txt

```diff
@@ -23,7 +23,10 @@ include(KDECompilerSettings NO_POLICY_SCOPE)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(Qt5 REQUIRED COMPONENTS Core)
+# global Qt version, used in all find_package calls
+set(OC_QT_VERSION 5.15 CACHE STRING "Qt version to build against" FORCE)
```

Comment:
There is no need to cache it.

Reply:
It shouldn't hurt, though. I'm not sure why I added the caching instructions. I can remove it.